ocs_ci/utility/gcp.py

```
@@ -75,6 +77,238 @@ def load_service_account_key_dict(filepath=SERVICE_ACCOUNT_KEY_FILEPATH):
+    #   workloadIdentityPools/<pool>/providers/<provider>
+    # We extract from gcp-ccm-cloud-credentials but any CCO secret works.
+    secret_ocp_obj = OCP(
+        kind=constants.SECRET,
+        namespace="openshift-cloud-controller-manager",
+        resource_name="gcp-ccm-cloud-credentials",
+    )
+    secret_data = secret_ocp_obj.get()["data"]
+
+    # The secret has a single data key containing the base64-encoded
+    # JSON credential file (typically "service_account.json")
+    encoded_creds = list(secret_data.values())[0]
```

Ruff (0.16.0)

[warning] Prefer `next(iter(secret_data.values()))` over single element slice

Replace with `next(iter(secret_data.values()))`

(RUF015)